sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -135,7 +135,25 @@ SELECT
 FROM
   base
 CROSS JOIN
+  -- See https://mozilla-hub.atlassian.net/browse/DENG-8432
+  -- Filtering out nimbus and normandy events that are emitted due to 'invalid-feature'.
+  -- The number of these events is too large to be processed, invalid-feature has also
+  -- been removed in more recent versions.
+  {% if app_name == "firefox_desktop_background_update" %}
+  ARRAY(
+    SELECT event
+    FROM UNNEST(events) AS event
+    WHERE NOT (
+      app_version_major = 138 AND app_version_minor IN (0, 1)
+      AND (
+        (event.category = 'nimbus_events' AND event.name = 'validation_failed')
+        OR (event.category = 'normandy' AND event.name = 'validation_failed_nimbus_experiment')
+      )
+    )
```

Comment:
I tried to be more selectively with the filtering here by testing expressions like:

```sql
AND EXISTS (
  SELECT 1
  FROM UNNEST(event.extra) AS kv
  WHERE kv.key = 'reason' AND kv.value = 'invalid-feature'
)
```

or 

```sql
ARRAY_LENGTH(
    ARRAY(
      SELECT 1
      FROM UNNEST(event.extra) AS kv
      WHERE kv.key = 'reason' AND kv.value = 'invalid-feature'
    )
  ) > 0
```
or 

```sql
mozfun.map.get_key(event.extra, "reason") = "invalid-feature"
```

But all of them result in resources being exceeded... Not sure if there is a more efficient way to only filtering these events.